week-3/E-array-map/exercise.js

```diff
@@ -4,7 +4,25 @@
 */
 
 // Complete this function -->
-function multiplyBy100(numbers) {}
+var numbers = [2, 3, 4, 5, 6];
+function multiplyBy100(number) {
+  // this the only one works in my labtop.
```

Comment:
If you can't get something to work, post a question in the class forum, saying what error you see.